src/tiler/package.json

```diff
@@ -4,8 +4,9 @@
   "description": "",
   "main": "server.js",
   "dependencies": {
-    "windshaft": "0.36.0",
-    "aws-sdk": "2.1.45"
+    "aws-sdk": "2.1.45",
+    "rollbar": "^0.5.12",
```

Comment:
Just making sure we want the carrot here. It may not matter since we have a shrinkwrap file.